docker/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Usage:
+# // To build and publish the Docker image to the internal container registry.
+# $ DESTINATION=internal INTERNAL_CONTAINER_REGISTRY=<internal_registry> ./release.sh
+# // To publish the Docker image to the public container registry.
+# $ DESTINATION=public ./release.sh
+set -ex
+
+PUBLIC_CONTAINER_REGISTRY='gcr.io/stackdriver-agents/stackdriver-logging-agent'
+
+source VERSION
+# TODO(qingling128): Remove this suffix when kubernetes-branch is deprecated.
+IMAGE_SUFFIX=""
```

Comment:
[Optional] Instead of making this a separate variable (and raising questions of how it'll be overridden), how about just changing the definition of `DOCKER_IMAGE_VERSION` on the `kubernetes` branch to
```
DOCKER_IMAGE_VERSION="${DOCKERFILE_VERSION}-${GOOGLE_FLUENTD_VERSION}-k8s"
```
?

Reply:
I'm really confused here. Why would you not simply modify this file on the `kubernetes` branch once this has been committed?